lasagne/layers/dnn.py

```diff
@@ -503,3 +502,95 @@ def convolve(self, input, **kwargs):
                                 conv_mode=conv_mode
                                 )
         return conved
+
+
+class SpatialPyramidPoolingDNNLayer(Layer):
+    """
+    lasagne.layers.SpatialPyramidPoolingDNNLayer(self, incoming,
+    pool_dims=[4,2,1], mode='max', **kwargs)
```

Comment:
Actually you can just omit this. We only need it when the default arguments include object instances like `lasagne.init.Uniform()`, because they would be rendered as `<lasagne.init.Uniform instance at 0x12345678>` by default.